Velaptor/UI/IWindow.cs

```diff
@@ -5,15 +5,120 @@
 namespace Velaptor.UI;
 
 using System;
+using System.Numerics;
 using System.Threading.Tasks;
+using Content;
+using Graphics.Renderers;
+using Scene;
 
 /// <summary>
 /// Provides the core of an application window which facilitates how the
 /// window behaves, its state and the ability to be used in various types
 /// of applications.
 /// </summary>
-public interface IWindow : IWindowActions, IWindowProps, IDisposable
+public interface IWindow : IDisposable
 {
+    /// <summary>
+    /// Gets or sets the <see cref="Action"/> delegate to be invoked one time to initialize the window.
+    /// </summary>
+    Action? Initialize { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Action"/> delegate that is invoked per frame for updating.
+    /// </summary>
+    Action<FrameTime>? Update { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Action"/> delegate that is invoked per frame for rendering.
+    /// </summary>
+    Action<FrameTime>? Draw { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Action"/> delegate to be invoked one time to uninitialize the window.
+    /// </summary>
+    Action? Uninitialize { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Action"/> delegate that is invoked every time the window is resized.
+    /// </summary>
+    Action<SizeU>? WinResize { get; set; }
+
+    /// <summary>
+    /// Gets or sets the title of the window.
+    /// </summary>
+    string Title { get; set; }
+
+    /// <summary>
+    /// Gets or sets the position of the window.
+    /// </summary>
+    Vector2 Position { get; set; }
+
+    /// <summary>
+    /// Gets or sets the width of the window.
+    /// </summary>
+    uint Width { get; set; }
+
+    /// <summary>
+    /// Gets or sets the height of the window.
+    /// </summary>
+    uint Height { get; set; }
+
+    /// <summary>
+    /// Gets or sets the value of how often the update and render calls are invoked in the value of hertz.
+    /// </summary>
+    int UpdateFrequency { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether or not the buffers should]
```

Comment:
Grammar (remove bracket)
```suggestion
    /// Gets or sets a value indicating whether or not the buffers should
```